content/2017.04.getting-started-with-elm.rst

```diff
@@ -50,32 +47,43 @@ another layer to setup, configure and maintain in your stack.
 .. _Flow: https://flow.org/en/
 .. _TypeScript: https://www.typescriptlang.org/
 
-Elm brings functional programming to your browser, the language is
-statically typed so with Elm you won't see any run-time errors caused
-by wrong type. Being purely functional, Elm lets you write code that
-is completely decoupled, which makes your code much more reusable and
+You can think of the Elm platform as the equivalent combined feature
+set of React_, Redux_, react-redux_, Babel_, eslint_, Flow/Typescript
+and Webpack_ all bundled in a single homogeneous and consistent package!
+
+.. _React: https://facebook.github.io/react/
+.. _react-redux: https://github.com/reactjs/react-redux
+.. _eslint: http://eslint.org/
+.. _Webpack: https://webpack.github.io/
+
+Elm brings functional programming to your browser. The language is
+statically typed, so with Elm you won't see any run-time errors caused
+by inconsistent typing. Being functional and pure, Elm let you write code
+that is completely decoupled, which makes your code more reusable and
+
 easier to refactor.
 
 
 How is Elm different?
 =====================
 
 Most of the time, using Elm, the two lines it takes to load your app
-is all the JavaScript you will write. Your application will be written
-in Elm and compiled into JavaScript.
+from the HTML file is all the JavaScript you will write, as your Elm
+code will be compiled to JavaScrit by the compiler.
 
-All the mistakes you make will be caught at compilation time, which
+Most of mistakes you will make will be caught at compilation time, which
 means that you will barely ever encounter exceptions at runtime in the
-browser.
+browser. Also that means less unit tests and defensive programming to
+write, which is always good.
 
-Because you will have a compilation step, Elm will force you to handle
-every possible case of your model state. It will make sure you covered
-them on the rendering side.
+Because there's a compilation step, Elm forces you to handle every
```

Comment:
I find it easier to read myself and I though it was a good writing principle to avoid `'s` but maybe that's not accurate.